plugin/src/test/groovy/com/novoda/staticanalysis/internal/ktlint/KtlintIntegrationTest.groovy

```diff
@@ -73,22 +97,22 @@ class KtlintIntegrationTest {
                 .build('evaluateViolations')
 
         assertThat(result.logs).containsKtlintViolations(1,
-                result.buildFileUrl("reports/ktlint/ktlint-${sourceSetName}.txt"))
+                result.buildFileUrl("reports/ktlint/$expectedOutputFileName"))
     }
 
     @Test
     void shouldNotFailBuildWhenNoErrorsEncounteredAndNoThresholdTrespassed() {
         def result = createProjectWith(Fixtures.Ktlint.SOURCES_NO_ERROR, 0)
-                .withToolsConfig(EMPTY_CONFIG)
-                .build('evaluateViolations')
+                .withToolsConfig(DEFAULT_CONFIG)
+                .build('evaluateViolations', '-s')
```

Comment:
I wonder if we should add `-s` to the default parameters instead 🤔

Reply:
That's a mistake :)